src/ol/expr/expression.js

```diff
@@ -240,6 +241,11 @@ export const Ops = {
 const parsers = {
   [Ops.Get]: createParser(AnyType, withArgsCount(1, 1), withGetArgs),
   [Ops.Var]: createParser(AnyType, withArgsCount(1, 1), withVarArgs),
+  [Ops.Concat]: createParser(
+    AnyType,
```

Comment:
```suggestion
    StringType,
```
Concat will always return string.

Reply:
Will apply with next commit with the compile expression move to outside the evaluation.